disa-lab/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "disa-lab",
+  "version": "1.0.0",
+  "description": "lab 22 JS401",
```

Comment:
missing .gitignore, let me know when you finish it so I can merge the PR, i think it looks cool btw